products/ocp4/profiles/cis-node.profile

```diff
@@ -25,7 +25,4 @@ description: |-
 
     This profile is applicable to OpenShift versions 4.10 and greater.
 
-filter_rules: '"ocp4-node" in platforms or "ocp4-master-node" in platforms or "ocp4-node-on-sdn" in platforms or "ocp4-node-on-ovn" in platforms'
```

Comment:
I build this content locally without the filter rules and the versionless profile had the same rules as the versioned profile, so it appears the filter is inherited through the usage of `extends`.